generators/ruby/faraday/templates/README.mustache

```diff
@@ -1,4 +1,4 @@
-# Onfido
+# Onfido Ruby Library
```

Comment:
Please update the first paragraph as in [python README.md](https://github.com/onfido/onfido-openapi-spec/blob/master/generators/python/urllib3/templates/README.mustache). Perhaps just the badges at the end of the paragraph:

```
# Onfido Ruby Library

The official Ruby library for integrating with the Onfido API.

Documentation can be found at <{{ documentationUrl }}>.

This version uses Onfido API {{ apiVersion }}. Refer to our [API versioning guide](https://developers.onfido.com/guide/api-versioning-policy#client-libraries) for details of which client library versions use which versions of the API.

[![Gem Version]...
```